# HG changeset patch
# User Rik <rik@octave.org>
# Date 1522684643 25200
#      Mon Apr 02 08:57:23 2018 -0700
# Branch stable
# Node ID 3166d47adde28d1e3a1852864a6ecfe7fff7c481
# Parent  d3baac355a2cc2c4d3d642c6e12b0a0388bbda93
RELEASE.PROCESS: Update checklist for making a release.

* RELEASE.PROCESS: Update checklist for making a release.

diff --git a/etc/RELEASE.PROCESS b/etc/RELEASE.PROCESS
--- a/etc/RELEASE.PROCESS
+++ b/etc/RELEASE.PROCESS
@@ -4,17 +4,17 @@
    on wiki.octave.org.
 <!----------------------------------------------------------------------------->
 ==4.4.0 Release Tasks==
 
 This page shows the tasks to be completed before the GUI release is finalized.
 
 <!----------------------------------------------------------------------------->
 # Update gnulib to latest version
-# Must occur first as it could resolve, or create new, bug reports
+# Must occur first as it could resolve existing, or create new, bug reports
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # File bug reports for all outstanding bugs known, but not reported
 #* Put out a general call for reports on Octave-Maintainers and Octave-Help list
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Review patch tracker/bug list for any patches submitted that may be included before release
 #: Completion Date:
@@ -25,30 +25,31 @@ This page shows the tasks to be complete
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Clear all bugs identified as must-fix
 #* See [[Bug Fix List - 4.2 Release]]
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # GPL License activities
 #* Update Copyright statements for all source controlled files
+#* Update dates in any other locations (launch message, citation, etc.)
 #* Add any new contributors to contributors.in
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Style-check code base
 #* This will produce lots of whitespace changes, but no behavior changes
 #* Must occur after patches have been added since whitespace changes will often prevent patches from applying
 #* m-file style check. Completion Date:
 #* C++ style check.  Completion Date:
 <!----------------------------------------------------------------------------->
 # Run lint checker on code base
-#* cppcheck, Clang sanitize, etc.
+#* cppcheck, -fsanitize, etc.
 #: Completion Date:
 <!----------------------------------------------------------------------------->
-# Verify 'make check' is passing on all buildbot combinations of OS and compiler
+# Verify 'make check' is passing on all buildbot combinations of OS and compilers
 #* Start discussion on octave-maintainers list about which failing tests must be fixed
 #* Identify and fix any tests determined critical in step above
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Compile and run Octave test suite with --enable-address-sanitizer-flags to check for memory leaks
 #* Results posted to bug report:
 #: Completion Date:
 <!----------------------------------------------------------------------------->
